### PR TITLE
YAML validation

### PR DESCRIPTION
**Pull request by tripleee**
This adds a test case to check the YAML lists for some simple problems.

Testing revealed a number of issues which are also solved with this PR.

- Embarrassing typos in a regex
- Embarrassing typos in a couple of watched NS entries
- Comment out a number of stale NSes which no longer resolve

The commented-out entries should probably be removed eventually, but some of the issues may be transient, so I want to keep them around and revisit them in a few weeks.

### CODE REVIEW

**Comment by tripleee** [comment]
The test currently adds about 45 seconds to the duration of the test suite. Maybe it should be hacked to run only at the end.

**Comment by tripleee** [comment]
The CI test failure looks spurious; I sometimes get this on my laptop, too.

Will rebase and re-push and hope it works this time.

File: watched_nses.yml [hunk 1]
```diff
@@ -516,7 +513,9 @@ items:
   ns: mihanwebhost.com.
 - ns: offshoreracks.com.
 - ns: sathyats.net.
-- ns: shared-host.org.
+- comment: no longert responding 2020-04-02
```
Comment (ArtOfCode-) [inline watched_nses.yml:516]: You've substituted typos in NSes to typos in comments :)

Comment (tripleee) [reply watched_nses.yml:516]: Thanks for noticing, I'll fix and merge on Monday.

**Comment by tripleee** [comment]
See also now #3793.

**Pull request merged**